test/specs/upload/conf/upload/schema.json

```diff
@@ -1,152 +1,299 @@
-            "tag:isrd.isi.edu,2017:asset": {
-              "url_pattern" : "/hatrac/ermrestjstest/{{{_fk_id}}}/{{{_uri.md5_hex}}}",
-              "filename_column" : "filename", 
-              "byte_count_column" : "bytes",
-              "md5" : "checksum"
-            }
-          }
+        "file_update_table": {
+            "comment": "table with 2 files to upload",
+            "kind": "table",
+            "keys": [
+                {
+                    "comment": null,
+                    "annotations": {},
+                    "unique_columns": [
+                        "key"
+                    ]
+                }
+            ],
+            "foreign_keys": [],
+            "table_name": "file_update_table",
+            "schema_name": "upload",
+            "column_definitions": [
+                {
+                    "name": "key",
+                    "nullok": false,
+                    "type": {
+                        "typename": "int"
+                    }
+                },
+                {
+                    "comment": null,
+                    "name": "file1_bytes",
+                    "default": null,
+                    "nullok": true,
+                    "type": {
+                        "typename": "int4"
+                    },
+                    "annotations": {}
+                },
+                {
+                    "comment": null,
+                    "name": "file1_MD5",
+                    "default": null,
+                    "nullok": true,
+                    "type": {
+                        "typename": "text"
+                    },
+                    "annotations": {}
+                },
+                {
+                    "comment": null,
+                    "name": "file1_name",
+                    "default": null,
+                    "nullok": false,
+                    "type": {
+                        "typename": "text"
+                    },
+                    "annotations": {
+                        "comment": [
+                            "hidden"
+                        ],
+                        "tag:misd.isi.edu,2015:display": {
+                            "name": "Thumbnail"
+                        }
+                    }
+                },
+                {
+                    "comment": "asset/reference",
+                    "name": "file1_uri",
+                    "default": null,
+                    "nullok": false,
+                    "type": {
+                        "typename": "text"
+                    },
+                    "annotations": {
+                        "comment": [
+                            "thumbnail",
+                            "download"
+                        ],
+                        "tag:isrd.isi.edu,2017:asset": {
+                            "byte_count_column": "file1_bytes",
+                            "url_pattern": "/hatrac/js/ermrestjs/{{{key}}}/{{{file1_MD5}}}",
```

Comment:
change this url_pattern to `/hatrac/js/ermrestjs/{{{_key}}}/{{{_file1_MD5}}}`. This means using the unformatted values for these two. It's true that the unformatted and formatted for the current values are the same, but if someone looks at this schema as an example, it might be misleading for them.